### PR TITLE
New project update project progression

### PR DESCRIPTION
**Pull request by MylesBorins**
This is part of https://github.com/openjs-foundation/bootstrap/pull/158

Specifically the changes to the already landed Project Progression document.

### CODE REVIEW

File: proposals/stage-1/PROJECT_ONBOARDING/README.md [hunk 1]
```diff
@@ -3,30 +3,51 @@
...
+Agreement on the initial checklist items; interim owners/helpers for each of the tasks.
 
+- [x] Template for Project Applications
+- [x] Onboarding Check List
+- [ ] Update to Project Progression
+- [ ] Potential changes to CPC Charter
+- [ ] Discussion/approval by bootstrap committee
+- [ ] Discussion/approval by the Board
+- [ ] Adoption by the CPC
+- [ ] Run existing projects through the process
```
Comment (mcollina) [inline proposals/stage-1/PROJECT_ONBOARDING/README.md]: What would be the desired outcome of  “run existing projects through the project”?

File: proposals/stage-1/PROJECT_ONBOARDING/README.md [hunk 2]
```diff
@@ -3,30 +3,51 @@
...
-We need to share a workflow and understanding of what steps are required to bring in projects to the OpenJS Foundation. This checklist is an attempt to organize our ideas and our work to make that happen effectively.
+We need to share a workflow and understanding of what steps are required to bring in projects to the OpenJS Foundation. This process is an attempt to organize our ideas and our work to make that happen effectively. This process should be explicit and consistent to make applications fair for all projects.
 
-## What is neccessary to complete this proposal
+## Unresolved Question
 
-Agreement on the initial checklist items; interim owners/helpers for each of the tasks.
+* Should there be a quiet period?
+* Should all emeritus projects be expected to go through this process?
+* Should projects be able to enter as `Impact Level`?
```
Comment (mcollina) [inline proposals/stage-1/PROJECT_ONBOARDING/README.md]: This is an important question. As long as the project meets all the criteria for an Impact level project, I do not see why not.
I think that considering certain projects “incubating” might not be something they would like to do.
Other projects might join in the “at large” category as well.

File: proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+- [ ] Update legal copyright notice on project website and github
+- [ ] Add OpenJS Foundation logo to project website
+- [ ] Add Project logo to OpenJS Foundation website; update PROJECTS.md file
+- [ ] Transfer logomark to the OpenJS Foundation
+- [ ] If project is using crowdfunding platforms, add disclaimer to platforms
+- [ ] Identify individuals from the project to join the CPC
+- [ ] Document project and foundation contacts for:
+  * marketing & social media
+  * infrastructure
+  * legal/governance help
```
Comment (mcollina) [inline proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md]: Does Legal need to do some provenance check or something?

**Comment by mcollina** [comment]
Why is this blocked?

**Comment by MylesBorins** [comment]
@mcollina it is blocked because it is a change to the landed Project Progression document. it shouldn't land until the associated proposal goes stage 3 (e.g. approved by bootstrap / CPC + board)

File: proposals/stage-1/PROJECT_ONBOARDING/README.md (hunk 1 above)
Comment (MylesBorins) [reply proposals/stage-1/PROJECT_ONBOARDING/README.md]: Can you moved this question to https://github.com/openjs-foundation/bootstrap/pull/158?

File: proposals/stage-1/PROJECT_ONBOARDING/README.md (hunk 2 above)
Comment (MylesBorins) [reply proposals/stage-1/PROJECT_ONBOARDING/README.md]: can you move this question to https://github.com/openjs-foundation/bootstrap/pull/158

File: proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md (hunk 3 above)
Comment (MylesBorins) [reply proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md]: can you move this question to https://github.com/openjs-foundation/bootstrap/pull/158

**Comment by mcollina** [comment]
To avoid confusion, we should close this PR and open a new one once #158 lands. Moving comments back and forth between PRs complicates things a little too much for me to handle.

File: PROJECT_PROGRESSION.md [hunk 4]
```diff
@@ -13,34 +13,16 @@ This proposal has been modified from the [CNCF process documentation](https://gi
...
- * The project will need to make these changes in order to progress further.
-* Projects get accepted via a 2/3 supermajority vote of the CPC.
-* The proposal document will be finalized as a project charter. This charter document must be included in the project's main repository.
-* The CPC will determine the appropriate initial stage for the project. The project can apply for a different stage via the review process. 
+
+New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a 2 week silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application projects will given an `incubation` status and begin the onboarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either volunatarily or at the request of the CPC.
```
[the remark this answers is not in the document]
Comment (mcollina) [reply PROJECT_PROGRESSION.md]: I don’t see this text in #158. I’m on mobile, so I might be missing things.

**Comment by MylesBorins** [comment]
@mcollina we should not close this PR it is part of #158 

I had originally made it so it was rebased on top of the other PR... I've gone ahead and rebased out the confusing commit... the changes in this PR are now completely self contained

File: PROJECT_PROGRESSION.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (MylesBorins) [reply PROJECT_PROGRESSION.md]: PTAL

File: PROJECT_PROGRESSION.md [hunk 5]
```diff
@@ -150,8 +132,28 @@ Projects in this stage are not in active development. Their maintainers may infr
...
+1. Projects that have been accepted into incubation process after silent period.
+
+**Expectations**
+
+It is expected that incubating project will make an active effort to work through the on-boarding process in a reasonable timeframe. They will be offered foundation support to help do so.
+
+**Acceptance Criteria**
+
+1. Completion of the on-boarding checklist
+1. 2/3 cote from the CPC to accept project
```
Comment (trivikr) [inline PROJECT_PROGRESSION.md]:
```suggestion
1. 2/3 supermajority vote from the CPC to accept project
```

File: PROJECT_PROGRESSION.md [hunk 6]
```diff
@@ -13,34 +13,16 @@ This proposal has been modified from the [CNCF process documentation](https://gi
...
+
+New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.
+
+#### Application Phases
+
+1. Initial email sent to new-projects@lists.openjsf.org with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).
+1. Silent period. Internal only discussion.
+1. Initial acceptance as [Incubating][] Project.
+1. Project goes through process of adhering to on-boarding checklist from [Project Application Template](./NEW_PROJECT_APPLICATION.md).
```
Comment (trivikr) [inline PROJECT_PROGRESSION.md]:
```suggestion
1. Project goes through process of adhering to onboarding checklist from [Project Application Template](./NEW_PROJECT_APPLICATION.md).
```

File: PROJECT_PROGRESSION.md [hunk 7]
```diff
@@ -13,34 +13,16 @@ This proposal has been modified from the [CNCF process documentation](https://gi
...
- * The project will need to make these changes in order to progress further.
-* Projects get accepted via a 2/3 supermajority vote of the CPC.
-* The proposal document will be finalized as a project charter. This charter document must be included in the project's main repository.
-* The CPC will determine the appropriate initial stage for the project. The project can apply for a different stage via the review process. 
+
+New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.
```
Comment (trivikr) [inline PROJECT_PROGRESSION.md]: This paragraph seems to be redundant, as steps are described in Application Phases.
How about we remove it partly/completely, and add info in phases instead?

Comment (MylesBorins) [reply PROJECT_PROGRESSION.md]: Some of the copy about incubation process isn't documented anywhere else. Perhaps that should be a separate section

Comment (sendilkumarn) [inline PROJECT_PROGRESSION.md]:
```suggestion
New projects should express interest to join the OpenJS Foundation via an email to <new-projects@lists.openjsf.org> with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. 
**Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.
```

File: PROJECT_PROGRESSION.md [hunk 8]
```diff
@@ -13,34 +13,16 @@ This proposal has been modified from the [CNCF process documentation](https://gi
...
+
+#### Application Phases
+
+1. Initial email sent to new-projects@lists.openjsf.org with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).
```
Comment (sendilkumarn) [inline PROJECT_PROGRESSION.md]:
```suggestion
1. Initial email sent to <new-projects@lists.openjsf.org> with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).
```

File: PROJECT_PROGRESSION.md (hunk 7 above)
Comment (sendilkumarn) [reply PROJECT_PROGRESSION.md]: Makes the e-mail clickable

**Comment by MylesBorins** [comment]
Rebased against master. This is necessary documentation for the project onboarding proposal and needs to land with https://github.com/openjs-foundation/cross-project-council/pull/248

@openjs-foundation/cpc PTAL

File: PROJECT_PROGRESSION.md [hunk 9]
```diff
@@ -157,7 +137,28 @@ Projects in this stage are not in active development. Their maintainers may infr
...
+
+1. Projects that have been accepted into incubation process after silent period.
+
+**Expectations**
+
+It is expected that incubating project will make an active effort to work through the on-boarding process in a reasonable timeframe. They will be offered foundation support to help do so.
+
+**Acceptance Criteria**
+
+1. Completion of the on-boarding checklist
```
Comment (mcollina) [inline PROJECT_PROGRESSION.md]: where is the on-boarding checklist defined? This should probably be a link.

File: PROJECT_PROGRESSION.md [hunk 10]
```diff
@@ -157,7 +137,28 @@ Projects in this stage are not in active development. Their maintainers may infr
 
 Projects may be granted Emeritus status via a 2/3 vote from the CPC and with approval from project ownership. In cases where there is a lack of project ownership, only a 2/3 vote from the CPC is required.
 
+### Incubating
+
+**Definition** 
+
+Incubating projects are projects that are in the process of completing their on-boarding checklist to join the foundation. Projects in this phase may looking to join the foundation as At-Large, Growth, or Impact Stage. Members of incubating projects are invited to join Cross Project Council meetings as a guest, please refer to the [CPC Charter](./CPC-CHARTER.md) for details on how guests may participate in the meetings.
```
Comment (timmywil) [inline PROJECT_PROGRESSION.md]: > Projects in this phase may looking

may be looking, I assume

**Comment by MylesBorins** [comment]
@mcollina can you please review, all requested changes should be reflected in the doc. This needs to land for https://github.com/openjs-foundation/cross-project-council/pull/248 to be able to land

**Pull request merged**